app/views/comment/_single_comment.html.erb

```diff
@@ -25,10 +25,12 @@
     <p class="event_actions">
       <% if comment.id %>
         <% if @user && @user.admin_page_links? %>
-          <%= link_to "Admin", edit_admin_comment_path(comment) %>
+          <%= link_to "Admin", edit_admin_comment_path(comment) %> |
         <% end %>
 
-        <!-- | <%= link_to _('Report abuse'), comment_path(comment) %> -->
+        <%= link_to _('Report abuse'),
```

Comment:
ah, css fiddling required to get it to look right as by default it does this:

![screen shot 2017-02-22 at 10 05 08](https://cloud.githubusercontent.com/assets/27760/23206674/6d499af0-f8e6-11e6-8a54-36b72cc8e65b.png)